docker-compose-separate-containers.yml

```diff
@@ -1,23 +1,25 @@
-version: '2'
+version: '3'
```

Comment:
No need to bump the version here, so that it can still work with older Docker and Docker Compose versions.